slurm/evaluate.slurm

```diff
@@ -5,6 +5,8 @@
 #SBATCH --output=./logs/%x-%j.out
 #SBATCH --error=./logs/%x-%j.err
 #SBATCH --requeue
+#SBATCH --time=1-00:00:00
```

Comment:
I don't think we have any evals that run longer than 1 day, but let me know if we should increase it